Packs/CommonScripts/Scripts/script-FailedInstances.yml

```diff
@@ -61,7 +61,8 @@ script: >
   var total = (countSuccess + countFailed).toString();
 
 
-  if (failedInstances.length === 0) {
+  if (countFailed === 0) {
+      failedInstances.push({})
```

Comment:
Please add a comment to explain the logic

Reply:
Done